.github/workflows/lab-server.yml

```
@@ -52,9 +52,9 @@ jobs:
       postgres:
         image: postgres
         env:
-          POSTGRES_USER: ${{ env.LAB_SERVER_DB_USER }}
-          POSTGRES_PASSWORD: ${{ env.LAB_SERVER_DB_PASS }}
-          POSTGRES_DB: ${{ env.LAB_SERVER_DB_NAME }}
+          POSTGRES_USER: ${{ env.DB_USER }}
+          POSTGRES_PASSWORD: ${{ env.DB_PASS }}
+          POSTGRES_DB: ${{ env.DB_NAME }}
```

```suggestion
          POSTGRES_USER: admin
          POSTGRES_PASSWORD: admin
          POSTGRES_DB: lab_server_db
```

Makes it consistent with the annotation server CI file.